install/cws-ui/deployments.ftl

```diff
@@ -9,6 +9,13 @@
 	<!-- Custom styles for this template -->
 	<link href="/${base}/css/dashboard.css" rel="stylesheet">
 	<script>
+	
+		//STATE PERSISTANCE CONSTS
+		const username = "username"; //temporary, hardcoded value for now
+		const lastNumHoursVar = "CWS_DASH_DEPLOY_LAST_NUM_HOURS" + username;
```

Comment:
Can you add like a "-" on the end of all these before the username?
Otherwise it would like like
CWS_DASH_DEPLOY_LAST_NUM_HOURSusername